.github/README.md

```diff
@@ -0,0 +1,27 @@
+# Bridge-In-Tech (android)
+
+
+## Contributing
+
+**This project is under active development**
+
+Bridge-In-Tech is an extension of Mentorship System that allows industry companies, mentors and students to actively contribute to one another. 
```

Comment:
> industry companies

can we put '/' in between these, coz they are actually similar. Also, make sure use all in 'plural' forms 😉 .
> contribute to

can we use 'collaborate with' instead?

Reply:
I also wouldn't say it is an extension of the mentorship system, the purpose is different than mentorship system. Right?
Also, this description should be directly below the title of the README, and not under Contributing header. Makes sense?